odoo/orm/fields.py

```diff
@@ -1194,6 +1194,9 @@ def _value_to_column(v):
         if operator in SQL_OPERATORS and isinstance(value, SQL):
             return SQL("%s%s%s", sql_field, SQL_OPERATORS[operator], value)
 
+        # nullability
+        nullable_column = not (self.required and model.env.registry.ready)
```

Comment:
`nullable_column` is not a column.  Also, for the sake of code readability, it would explicitly introduce its inverse
```suggestion
        can_be_null = not (self.required and model.env.registry.ready)
```